example/heatEquation2D/src/analyticalSolution.hpp

```diff
@@ -0,0 +1,76 @@
+/* Copyright 2020 Tapish Narwal
+ * SPDX-License-Identifier: ISC
+ */
+
+#pragma once
+
+#include <alpaka/alpaka.hpp>
+
+#include <cmath>
+
+//! Exact solution to the test problem
+//! u_t(x, y, t) = u_xx(x, t) + u_yy(y, t), x in [0, 1], y in [0, 1], t in [0, T]
+//! u(0, t) = u(1, t) = 0
+//! u(x, 0) = sin(pi * x)
+//! u(0, y) = sin(pi * y)
+//!
+//! \param x value of x
+//! \param x value of y
+//! \param t value of t
+
+ALPAKA_FN_HOST_ACC auto exactSolution(double const x, double const y, double const t) -> double
```

Comment:
```suggestion
ALPAKA_FN_HOST_ACC auto exactSolution(double const t, double const x, double const y) -> double
```
is what I would have written. That is also spiritually closer to the data layout isn't it? (I'm aware that we only hold slices for two t at any given time but it's still somehow the "slowest index".